### DISCUSSION TITLE
ERROR - Exception

### DISCUSSION THREAD

**Discussion by niteowlcreative** [statamic/cms#4546] [Troubleshooting] [answered]
For all pages im seeing:

Declaration of Statamic\Tags\ArrayAccessor::hasAny(array $keys) should be compatible with Illuminate\Support\Collection::hasAny($key)

Do we know why this is an error? I am unsure how to resolve.

**Comment by jasonvarga** [comment, marked as answer]
This was recently fixed. Upgrade Statamic.

**Reply by niteowlcreative** [reply]
I did upgrade. 



composer update statamic/cms --with-dependencies

> Statamic\Console\Composer\Scripts::preUpdateCmd
Loading composer repositories with package information
Dependency "guzzlehttp/guzzle" is also a root requirement. Package has not been listed as an update argument, so keeping locked at old version. Use --with-all-dependencies (-W) to include root dependencies.
Dependency "laravel/framework" is also a root requirement. Package has not been listed as an update argument, so keeping locked at old version. Use --with-all-dependencies (-W) to include root dependencies.
Updating dependencies
Nothing to modify in lock file
Installing dependencies from lock file (including require-dev)
Nothing to install, update or remove
Package sebastian/resource-operations is abandoned, you should avoid using it. No replacement was suggested.
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi
Discovered Package: aryehraber/statamic-captcha
Discovered Package: barryvdh/laravel-debugbar
Discovered Package: facade/ignition
Discovered Package: fideloper/proxy
Discovered Package: fruitcake/laravel-cors
Discovered Package: intervention/image
Discovered Package: laravel/tinker
Discovered Package: nesbot/carbon
Discovered Package: nunomaduro/collision
Discovered Package: rebing/graphql-laravel
Discovered Package: statamic/cms
Discovered Package: wilderborn/partyline
Package manifest generated successfully.
> @php artisan statamic:install --ansi
Discovered Addon: aryehraber/statamic-captcha
Addon manifest generated successfully.
Copied Directory [/vendor/statamic/cms/resources/users] To [/resources/users]
Publishing complete.
Copied Directory [/vendor/statamic/cms/resources/dist] To [/public/vendor/statamic/cp]
Publishing complete.
Compiled views cleared!
Application cache cleared!
99 packages you are using are looking for funding.
Use the `composer fund` command to find out more!

**Reply by jasonvarga** [reply]
What `statamic/cms` constraint is used in your `composer.json`?

**Reply by niteowlcreative** [reply]
im not sure what this means.

**Reply by duncanmcclean** [reply]
What does the line that says `statamic/cms` look like in your `composer.json` file?

**Reply by niteowlcreative** [reply]
"statamic/cms": "3.1.*"

**Reply by jasonvarga** [reply]
You need to upgrade to 3.2.
https://statamic.dev/upgrade-guide/3-1-to-3-2

**Reply by niteowlcreative** [reply]
Thank you both. That worked.
